server-common/src/main/java/org/apache/cassandra/sidecar/common/server/cluster/locator/TokenRange.java

```diff
@@ -75,18 +75,83 @@ public TokenRange(long start, long end)
 
     public TokenRange(BigInteger start, BigInteger end)
     {
-        this.start = start;
-        this.end = end;
+        this(Token.from(start), Token.from(end));
```

Comment:
Should we validate that end is bigger than start?

Reply:
This should already be taking care by `Range.openClosed`